website/docs/r/log_analytics_workspace.html.markdown

```diff
@@ -43,6 +43,8 @@ The following arguments are supported:
 
 * `retention_in_days` - (Optional) The workspace data retention in days. Possible values are either 7 (Free Tier only) or range between 30 and 730.
 
+* `daily_quota_gb` - (Optional) The workspace of the volume cap in gb.  Defaults to -1 (unlimited).
```

Comment:
Can we adjust the statement a bit?

```suggestion
* `daily_quota_gb` - The workspace daily quota for ingestion in GB.  Defaults to -1 (unlimited).
```

Reply:
Do we not want the `(Optional)` in the resource?

Reply:
we do